crates/solidity-v2/outputs/cargo/parser/src/lexer/tests/mod.rs

```diff
@@ -18,7 +18,7 @@ fn test_lexer_output(
     version: Version,
     kind: ContextKind,
 ) {
-    let mut lexer = Lexer::new(kind, source, version);
+    let mut lexer = Lexer::new(kind, source, version.try_into().unwrap());
```

Comment:
Given that this is a V2 util, should we define/pass `LanguageVersion` directly from callers?